src/components/base/text/Text.module.css

```diff
@@ -22,26 +23,113 @@
   font-size: 14px;
   line-height: 22px;
 }
+
 .text4 {
   font-size: 14px;
   line-height: 22px;
   font-family: var(--body-font-semibold);
 }
+
 .text5 {
   font-size: 11px;
   line-height: 17px;
 }
+
 .text6 {
   font-size: 11px;
   line-height: 17px;
   font-family: var(--body-font-semibold);
 }
+
 .clamp {
   display: -webkit-box;
   -webkit-box-orient: vertical;
   overflow: hidden;
 }
 
+/* Titles */
+.title {
+  font-family: var(--title-font);
+
+  /* Bootstrap default override */
+  margin: 0;
+}
+
+.title1 {
+  font-size: 32px;
```

Comment:
her kunne man bruge vars i stedet for pixels

Reply:
Kunne man godt på nogen. Men tænker faktisk ikke det burde bindes sammen. 

1. fonts har mange variabler uden for vores spacings.
2. Hvis vi ville modificere vores spacings, vil vi ikke nødvendigvis tilpasse font str. Så bindingen giver ikke nødvendigvis mening. Ku være hvis man ønskede mere/mindre luft på sitet.

Reply:
Jeg ved ikke om Kim mener det samme som jeg mener, men jeg synes det er vigtigt at vi har custom properties til hver skriftstørrelse + line height. F.eks. `--text1_font_size`, `--title3_line_height` (eller med dromedarCase hvis I foretrækker det).

Det vil være nice at få på plads når du er i gang med disse forbedringer, for det vil være nice hvis det er muligst at bruge f.eks. `var(--title3_font_size)` og `var(--title3_line_height)` i andre komponenter, uden at skulle hardcode det...